metals/src/main/scala/scala/meta/internal/metals/ImportedBuild.scala

```diff
@@ -43,11 +46,35 @@ case class ImportedBuild(
 }
 
 object ImportedBuild {
+  import scala.meta.internal.metals.MetalsEnrichments._
```

Comment:
Can we move it up to the other imports? That's usually how it's done in other places.

Reply:
Well I purposefully put it here since in the actual class we do a lot of `.asScala` in various places, and when we put the `MetalsEnrichments` in scope of those, everything breaks. So this was an easy way to not have to touch any of that code and only scope it to what is using it here. Is that alright, or do you want me to also refactor the `++` and `targetUris` in `ImportedBuild`?

Reply:
Actually nevermind. It was just conflicting with one of the other imports. I can just remove it and all is good.